BitBox02 implement (#12341)
zkSNACKs/WalletWasabi: WalletWasabi.Documentation/WasabiCompatibility.md

# Officially Supported Hardware Wallets

> - Blockstream Jade
> - BitBox02-BtcOnly

Reviewer: ```suggestion
- BitBox02-BtcOnly
- Blockstream Jade
```

- Ledger Nano X
- Trezor Model T
- Blockstream Jade

> - BitBox02

Reviewer 2: Not directly related to this PR, but as there are more and more signing devices supported, wouldn't it make sense to sort this list alphabetically?

Author: thats right. okey i will

Reviewer 3: Write the only one that we are surely supporting.

- Ledger Nano S Plus
- Ledger Nano X
- Trezor Model T

> 

Reviewer: ```suggestion
```

- Ledger Nano S Plus
- Ledger Nano X
- Trezor Model T

> <sup><sup>1*</sup> The device by default asks for a "Pairing code", currently, there is no such function in WW. Therefore, either disable the feature or unlock it with BitBoxApp or hwi-qt before using the program.</sup>

Reviewer 3: ```suggestion
<sup><sup>1*</sup> The device by default asks for a "Pairing code", currently, there is no such function in Wasabi. Therefore, either disable the feature or unlock the device with BitBoxApp or hwi-qt before using it with Wasabi.</sup>
```
